docs/source/operationalguide.rst

```diff
@@ -20,5 +20,6 @@ This section defines the pre-requisites installation and steps for setup of a DL
     operations/upgrade_fabricnetwork.md
     operations/adding_new_org_fabric
     operations/indy_verify.md
+    operations/fabric_verify.md
```

Comment:
The verify.md file is missing

Reply:
This will be added in another pull request.

Reply:
Seperate request for story 882

Reply:
then remove this line and merge into single commit. Everything else is good